dali/ft/filecopy.cpp

```diff
@@ -592,6 +596,24 @@ FileSprayer::FileSprayer(IPropertyTree * _options, IPropertyTree * _progress, IR
     encryptKey.set(options->queryProp(ANencryptKey));
     decryptKey.set(options->queryProp(ANdecryptKey));
 
+    fileUmask = -1;
+    if (options->hasProp(ANumask))
+    {
+        StringBuffer umaskStr;
+        options->getProp(ANumask, umaskStr);
+        errno = 0;
+        fileUmask = (int)strtol(umaskStr.str(), NULL, 8);
+        if (errno)
```

Comment:
I could use std::stoi and check for std::invalid_argument exception but that requires c++11 I think? so ok for 6.0.0 but no good for 5.6.4.